### ISSUE TITLE
Add-on Crashes Kodi while scanning Music

### ISSUE THREAD

**Issue by charredchar** [jellyfin/jellyfin-kodi#225]
I've been using the Jellyfin Plugin for Kodi for a little while now and this seemed to "happen more and more" as time went on, I could not pinpoint a specific cause. I eventually completely wiped my HTPC and started from scratch but this bug followed, crashing constantly after installing the plugin. I eventually narrowed it down to a crash happening during the scan of my Music library, it could have been crashing while I scroll past Music on the home screen too.

The most recent being a fresh install,
Install Add-on,
Select your server,
Native directories,
Select Libraries,
Let it scan.

Sorry.. I do not have a log now since I managed to get it working by not selecting the Music library and this is my main HTPC that I need up and running. I should note that I selected the music share directly in Kodi and it is working as it should, not a huge loss to me because I don't need to sync my Music status between everything. I will set up a VM shortly to collect logs, it could as easily have something to do with character sets as I do have non-english titles on some music.

 - OS: Windows 10 Enterprise LTSC 1809
 - Jellyfin Version: 10.4.1
 - Kodi Version: 18.5
 - Addon Version: 0.5.0

**Comment by mcarlton00** [comment]
I have my music synced, but I haven't noticed this problem.  Logs will be incredibly helpful towards tracking it down.  How large is your music library?

**Comment by charredchar** [comment]
That is understandable, sorry there was a delay posting it. The plus side is I ruled out the Music being the issue as Kodi started crashing once again even without the Music library in Kodi at all. This gets so bad that it will crash within seconds of logging in, I have to uninstall the Jellyfin Add-on from the master user before I can get into my profile and remove the sources and try again. As a test I also did not reinstall the Jellyfin Add-on, I instead added the shares directly into Kodi to verify this was for sure not only a Kodi issue. And guess what.. crashed anyway. I'll take the issue over to the Kodi forums first but post the logs here just in case anyone wants to take a crack as it.

https://paste.ubuntu.com/p/74g4G4c7xG/
